app/controllers/users_controller.rb

```diff
@@ -1,32 +1,79 @@
 class UsersController < ApplicationController
-  before_action :userprofile, only: :show
+  before_action :logged_in_user, only: %i(index edit update destroy)
+  before_action :load_user, only: %i(edit update destroy show)
```

Comment:
![image](https://user-images.githubusercontent.com/65896888/91373274-2e444b00-e840-11ea-8176-6193ecbd2de0.png)

Reply:
I cannot liad the sign in page and log in  error message after i change only to except